internal/bminventory/inventory_test.go

```diff
@@ -9340,6 +9351,46 @@ var _ = Describe("V2DownloadInfraEnvFiles", func() {
 		Expect(lines[3]).To(Equal("boot"))
 	})
 
+	It("returns ipxe-script successfully - with kernel arguments", func() {
+		kernelArguments := []string{
+			"p1",
+			"p2=argument",
+			"p3=\"this is an argument\"",
+		}
+		initialKernelArguments := `random.trust_cpu=on rd.luks.options=discard ignition.firstboot ignition.platform.id=metal console=tty1 console=ttyS1,115200n8 coreos.inst.persistent-kargs="console=tty1 console=ttyS1,115200n8"`
+		mockVersions.EXPECT().GetOsImageOrLatest(common.TestDefaultConfig.OpenShiftVersion, gomock.Any()).Return(common.TestDefaultConfig.OsImage, nil).Times(3)
+		content := getResponseData("ipxe-script", false, nil, "")
+		lines := strings.Split(string(content), "\n")
+
+		Expect(lines[0]).To(Equal("#!ipxe"))
+
+		By("validating the kernel line")
+		kernelRegex := regexp.MustCompile(`^kernel (\S+) initrd=initrd coreos.live.rootfs_url=(\S+) (.+)`)
+		match := kernelRegex.FindStringSubmatch(lines[2])
+		Expect(match).NotTo(BeNil())
+
+		Expect(match[3]).To(Equal(initialKernelArguments))
```

Comment:
I don't think you need to test the initial args as we're already testing that in other places.

Reply:
OK

Reply:
It just doesn't seem helpful to me to have two tests fail for exactly the same thing if/when we change the default set of params, but it's fine if you want to keep it.

Reply:
OK